keps/sig-storage/3476-volume-group-snapshot/README.md

```diff
@@ -437,10 +487,11 @@ Type VolumeGroupSnapshotContentStatus struct {
         // +optional
         Error *VolumeSnapshotError
 
-        // List of volume snapshot content refs
-        // The max number of snapshots in a group is 100
-        // +optional
-        VolumeSnapshotContentRefList []core_v1.ObjectReference
+	// PVVolumeSnapshotContentList is the list of pairs of PV and
+	// VolumeSnapshotContent for this group snapshot
+	// The maximum number of allowed snapshots in the group is 100.
+	// +optional
+	PVVolumeSnapshotContentList []PVVolumeSnapshotContentPair
```

Comment:
See the previous comment, I think we either don't need to have this array, or we need to rework it to couple snapshot handle with the source PV / source volume handle, so the controller can create VolumeSnapshots and VolumeSnapshotContents.

Reply:
PVVolumeSnapshotContentList were needed previously because csi-snapshotter sets them after creating individual objects. The group snapshot controller watches that and sets PVCVolumeSnapshotRefList based on that. 
With the latest change, it looks like we don't need it any more. I'd like to keep it here for now and we can remove it at implementation time with more testing.

Reply:
Can you add a comment saying that we will consider removing this field after testing.

Reply:
Sure, I added the comment.